docs/example/searchbarDocs.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <doc-section
+    id="searchbar"
+    name="searchbar">
+    <div class="bs-example">
+      <searchbar
+        :data="searchData"
+        :template="searchTemplate"
+        template-name="search"
+      />
+    </div>
+    <doc-code language="markup">
+      <searchbar
+        :data="searchData"
+        :template="searchTemplate"
+        template-name="search"
+      />
+    </doc-code>
+    <!-- eslint-disable -->
+    <doc-code language="javascript">
+    new Vue {
+      components: {
+        searchbar
+      },
+      data() {
+        return {
+          searchData: [
+            {
+              'headings': {
+                'normal-include': 'Normal include',
+                'establishing-requirements': 'Establishing Requirements',
+                'include-segment': 'Include segment',
+                'dynamic-include': 'Dynamic include',
+                'boilerplate-include': 'Boilerplate include',
+                'nested-include': 'Nested include',
+                'html-include': 'HTML include',
+                'include-from-another-markbind-site': 'Include from another Markbind site',
+                'feature-list': 'Feature list',
+              },
+              'title': 'Hello World',
+              'src': 'index.md',
+            },
+            {
+              'headings': {
+                'popover-initiated-by-trigger-honor-trigger-attribute':
+                'popover initiated by trigger: honor trigger attribute',
+                'support-multiple-inclusions-of-a-modal': 'Support multiple inclusions of a modal',
+                'remove-extra-space-in-links': 'Remove extra space in links',
+              },
+              'title': 'Open Bugs',
+              'src': 'bugs/index.md',
+            },
+            {
+              'headings': {
+                'feature-list': 'Feature list',
+              },
+              'src': 'sub_site/index.md',
+              'title': '',
+            },
+          ],
+          searchTemplate: '<span v-html="item.title | highlight value" /><br />'
+            + '<span v-if="item.keywords" v-html="item.keywords | highlightAndExtract value<br />" />'
+            + '<span v-if="item.heading" v-html="item.heading.text | highlight value" />',
+        };
+      }
+    }
+    </doc-code>
+    <!-- eslint-disable -->
+  </doc-section>
```

Comment:
Do provide a table describing each attributes. See how the original developer did it at the bottom of the [Typehead](https://yuche.github.io/vue-strap/#typeahead) section.